truenas/webui
# NAS-114098 / None / NAS-114098: Expose configurable options when deleting VM device in UI

Testing:

1. Create one or more VMs. Keep auto-created ZVol which is default hard-drive settings
2. Open Devices for each VM, and add "RAW File" device
3. Choose "Delete device" from device menu
   - Try deleting Disks with and without ZVol option. Check that Zvols actually removed (can be checked by viewing Storage page)
   - Try deleting Raw files with and without Raw file option
   - Try deleting with and without Force option
